src/lib/core/authorization/AuthorizationManager.ts

```diff
@@ -319,11 +321,26 @@ export class AuthorizationManager {
    * This method will introspect the response and attempt to handle any errors that should result
    * in some additional Globus Auth interaction.
    * @param response - The error response from a Globus service.
-   * @param execute - Whether to execute the handler immediately or return a function that can be executed later.
+   * @param options - Options for handling the error response.
+   * @param options.execute - Whether to execute the handler immediately.
+   * @param options.overrides - Overrides for the handler.
    */
-  handleErrorResponse(response: Record<string, unknown>, execute?: true): void;
-  handleErrorResponse(response: Record<string, unknown>, execute?: false): () => void;
-  handleErrorResponse(response: Record<string, unknown>, execute = true) {
+  handleErrorResponse(
+    response: Record<string, unknown>,
+    options?: { execute: true; additionalParams?: IObject } | true,
+  ): void;
+  handleErrorResponse(
+    response: Record<string, unknown>,
+    options?: { execute: false; additionalParams?: IObject } | false,
+  ): () => void;
+  handleErrorResponse(
+    response: Record<string, unknown>,
+    options?: { execute: boolean; additionalParams?: IObject } | boolean,
```

Comment:
The `options` type here should be `{ execute?: boolean; additionalParams?: IObject }`; `handleErrorResponse(res, { additionalParams: { foo: 'bar'} });` should be a valid call.

Reply:
ah yeah, good call